tools/windows-node-installer/pkg/commandline/flags.go

```diff
@@ -1,9 +1,9 @@
 package commandline
 
-// InstallMode defines functionalities supported by the windows-node-installer
-type InstallMode string
+// Installmode defines functionalities supported by the windows-node-installer
```

Comment:
`InstallMode` was correct.

Reply:
Sorry, I might have misinterpreted @ravisantoshgudimetla's comment.

Reply:
He is talking about type casting in https://github.com/openshift/windows-machine-config-operator/pull/6#discussion_r319347271 and not naming.